pkg/kube/vault/vault_test.go

```diff
@@ -43,7 +43,7 @@ func TestCreateVault(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			err := vault.CreateVault(client, vaultclient, tc.name, tc.namespace, tc.gcpSecretName,
+			err := vault.CreateGKEVault(client, vaultclient, tc.name, tc.namespace, tc.gcpSecretName,
```

Comment:
It would be nice to have a test also for AWS.

Reply:
I'll see what I can do. I'll first push the above fixes.

Reply:
I have now pushed an equivalent test for AWS